wiki/Modding/Nomination/en.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - nominations
+  - nom
+  - noms
+  - heart
+---
+
+# Nomination
+
+*See also: [Beatmap ranking procedure](/wiki/Beatmap_ranking_procedure)*
+
+::: Infobox
+![](img/nomination-progress-beatmapset1857663.png "Beatmap ranking status displayed on the discussion page, with hype and nomination progress bars partially filled.")
+:::
+
+A **nomination** is a formal vote of approval coming from [beatmap nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [ranking criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).
+
+Beatmaps that receive two nominations per each available [game mode](/wiki/Game_mode) are "[qualified](/wiki/Beatmap/Category#qualified)" and enter the [beatmap ranking queue](/wiki/Beatmap_ranking_procedure/Ranking_queue), where they stay for at least 7 days for further inspection. If no critical issues are found, beatmaps are then moved to the Ranked category and receive their own [leaderboards](/wiki/Ranking).
+
+Nominations may be removed from a map in one of the following cases:
+
+- A beatmap is updated by its [host](/wiki/Beatmap/Beatmap_host).
+- Serious or unrankable issues are found by a BN or a member of the NAT.
```

Comment:
could add this if you want people to know the 3 kinds of nomination resets there are
```suggestion
- Serious or unrankable issues are found by a BN or a member of the NAT, which would cause them to either [reset its nomination](/wiki/People/The_Team/Beatmap_Nominators/General_Information#disqualify-or-reset-nomination), [disqualify](/wiki/People/The_Team/Beatmap_Nominators/General_Information#disqualify-or-reset-nomination) it, or place a [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto).
```

Reply:
this is valuable, thanks